_data/guides.yaml

```diff
@@ -7,6 +7,9 @@ categories:
       - title: Getting Started with WildFly
         url: /get-started
         description: Build and run a Jakarta EE application with WildFly in a few minutes
+      - title: Prototyping WildFly Applications with JBang
```

Comment:
This and the next comment are about shortening this so the space is takes on the guides page is closer to its neighbor.

"Prototyping with WildFly"

The next bit says "WildFly applications" so that's covered.